phoenix-core/src/it/java/org/apache/phoenix/end2end/SkipScanAfterManualSplitIT.java

```diff
@@ -43,7 +43,7 @@
 import com.google.common.collect.Maps;
 
 
-public class SkipScanAfterManualSplitIT extends BaseHBaseManagedTimeIT {
+public class SkipScanAfterManualSplitIT extends BaseHBaseManagedTimeTableReuseIT {
```

Comment:
I think table names can be randomized here. I would also get rid of tableName and tableNameBytes static constants.